tests/unit/test_tuples_iter.cpp

```diff
@@ -373,4 +373,67 @@ TEST_F(TuplesTest, ColumnIteratorEmptyMatrixTest) {
 
 	GxB_MatrixTupleIter_free(iter);
 	GrB_Matrix_free(&A);
-}
\ No newline at end of file
+}
+
+TEST_F(TuplesTest, IteratorJumpToRowTest) {
+
+	// Matrix is 5X5 and will be populated with the following indices.
+	GrB_Index indices[5][2] = {
+		{0, 2},
+		{2, 1},
+		{2, 3},
+		{3, 0},
+		{3, 4}
+	};
+
+	bool depleted;
+	GrB_Info info;
+	GrB_Index row;
+	GrB_Index col;
+
+	// Create and populate the matrix.
+	GrB_Index n = 5;
+	GrB_Matrix A = CreateSquareNByNEmptyMatrix(n);
+	for(int i = 0; i < 5; i ++) {
+		row = indices[i][0];
+		col = indices[i][1];
+		GrB_Matrix_setElement_BOOL(A, true, row, col);
+	}
+
+	// Create iterator.
+	GxB_MatrixTupleIter *iter;
+	GxB_MatrixTupleIter_new(&iter, A);
+
+	// Check for invalid index exception for row jump.
+	info = GxB_MatrixTupleIter_jump_to_row(iter, -1);
+	ASSERT_EQ(GrB_INVALID_INDEX, info);
+	info = GxB_MatrixTupleIter_jump_to_row(iter, n);
+	ASSERT_EQ(GrB_INVALID_INDEX, info);
+
+	// Check for legal jump to row, and retrive row's value.
+	info = GxB_MatrixTupleIter_jump_to_row(iter, 2);
+	ASSERT_EQ(GrB_SUCCESS, info);
+
+	// Check that the right indices are retrived.
+	for(int i = 1; i < 3; i++) {
+		info = GxB_MatrixTupleIter_next(iter, &row, &col, &depleted);
+		ASSERT_EQ(GrB_SUCCESS, info);
+		ASSERT_EQ(indices[i][0], row);
+		ASSERT_EQ(indices[i][1], col);
+		ASSERT_FALSE(depleted);
+	}
```

Comment:
row 2 and 3 has 4 entires in total:
```{2, 1}, {2, 3}, {3, 0}, {3, 4}```
I think this loop should perform 4 iterations

Reply:
once all 4 iterations are done perform another iteration and check that it is depleted.